prow/scripts/provision-vm-and-start-kyma.sh

```diff
@@ -24,15 +24,52 @@ else
     LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=kyma-integration")
 fi
 
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+
+    key="$1"
+
+    case ${key} in
+        --image)
+            IMAGE="$2"
+            shift
+            shift
+            ;;
+        --*)
+            echo "Unknown flag ${1}"
+            exit 1
+            ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+            ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+if [[ -z "$IMAGE" ]]; then
+    shout "Provisioning vm using the latest default custom image ..."   
+    
+    DEFAULT_IMAGES=$(gcloud compute images list --sort-by "~creationTimestamp" \
+         --filter "family:custom images AND labels.default:yes" | tail -n +2 | awk '{print $1}')
+
+    default_image_arr=($DEFAULT_IMAGES)
+    IMAGE=${default_image_arr[0]}
```

Comment:
We can simplify this, just add `--limit=1` to the `DEFAULT_IMAGES` call :)

Reply:
The code has been updated accordingly, --limit=1 is used now